packages/react/src/controls/dropdowns/FieldAggregateOperationDropdown.tsx

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) Microsoft. All rights reserved.
  * Licensed under the MIT license. See LICENSE file in the project.
  */
-import { FieldAggregateOperation } from '@data-wrangling-components/core'
+import { FieldAggregateRollupOperation } from '@data-wrangling-components/core'
```

Comment:
You can keep the name of this as `FieldAggregateOperation` since aggregate/rollup basically mean the same thing